motor/discovery/aws/ec2_instances.go

```diff
@@ -183,11 +215,11 @@ func instanceToAsset(account string, region string, instance types.Instance, ins
 		}
 	}
 	// add passed in labels
-	for k, v := range passInLabels {
+	for k, v := range instanceInfo.passInLabels {
 		asset.Labels[k] = v
 	}
 	// add AWS metadata labels
-	asset.Labels = addAWSMetadataLabels(asset.Labels, ec2InstanceToBasicInstanceInfo(instance, region, account))
+	asset.Labels = addAWSMetadataLabels(asset.Labels, ec2InstanceToBasicInstanceInfo(instanceInfo.instance, instanceInfo.region, instanceInfo.account))
```

Comment:
should we just accept the in `instanceInfo` as input for `ec2InstanceToBasicInstanceInfo`?